packages/manager/src/utilities/formikErrorUtils.test.ts

```diff
@@ -201,3 +202,229 @@ describe('getFormikErrorsFromAPIErrors', () => {
     }
   });
 });
+
+describe('Tests for set', () => {
+  it("returns the passed in 'object' as is if it's not actually a (non array) object", () => {
+    expect(set([], 'path not needed', 3)).toEqual([]);
+  });
+
+  describe('Correctly setting the value at the given path', () => {
+    it('sets the value for a simple path for both string and array paths', () => {
+      const object = {};
+      let settedObject = set(object, 'test', 1);
+      expect(object).toBe(settedObject);
+      expect(object).toEqual({ test: 1 });
+
+      settedObject = set(object, 'test2', 1);
+      expect(object).toBe(settedObject);
+      expect(object).toEqual({ test: 1, test2: 1 });
+    });
+
+    it('sets the value for complex string and array paths (without indexes)', () => {
+      const object = {};
+
+      // the given paths are equivalent in string vs array format
+      set(object, 'a.b.c', 'c');
+      expect(object).toEqual({ a: { b: { c: 'c' } } });
+
+      set(object, 'a.b.d', 'd');
+      expect(object).toEqual({
+        a: { b: { c: 'c', d: 'd' } },
+      });
+
+      set(object, 'e[f][g]', 'g');
+      expect(object).toEqual({
+        a: { b: { c: 'c', d: 'd' } },
+        e: { f: { g: 'g' } },
+      });
+    });
+
+    it('sets the value for complex string and array paths (with indexes)', () => {
+      const object = {};
+
+      // the given paths are equivalent in string vs array format
+      set(object, 'a.b.1', 'b1');
+      expect(object).toEqual({ a: { b: [undefined, 'b1'] } });
+      set(object, 'a.b[0]', 5);
+      expect(object).toEqual({ a: { b: [5, 'b1'] } });
+
+      // If path is an array, indexes can be passed in as a string or as a number
+      set(object, 'a.b.2', 'b2');
+      expect(object).toEqual({
+        a: { b: [5, 'b1', 'b2'] },
+      });
+
+      set(object, 'a.b[3].c', 'c');
+      expect(object).toEqual({
+        a: { b: [5, 'b1', 'b2', { c: 'c' }] },
+      });
+    });
+
+    it.skip('creates an empty string key', () => {
+      expect(set({}, '', 'empty string')).toEqual({ '': 'empty string' });
+    });
+
+    it.skip('only considers valid indexes for setting array values', () => {
```

Comment:
different behavior for these tests

(quick change can be made to get test @ line 267 working if we want)